client/src/components/Feed/FiltersSidebar.js

```diff
@@ -0,0 +1,137 @@
+// Core
+import React, { useContext } from "react";
+import styled from "styled-components";
+
+// Antd 
+import { Drawer } from 'antd';
+import { Button } from "antd-mobile";
+
+// Local
+import FilterAccordion from "./FilterAccordion";
+import SubmitButton from "components/Button/SubmitButton";
+import { FeedContext } from "pages/Feed";
+
+// Constants
+import { theme } from "constants/theme";
+import { ROYAL_BLUE } from "constants/colors";
+
+// Icons
+import { ReactComponent as BackIcon } from "assets/icons/back.svg";
+
+const DrawerWrapper = styled(Drawer)`
+  .ant-drawer-body {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 20px;
```

Comment:
Let's use `rem` values for all the `px` values. Right now 1rem = 10px

Reply:
Done.